tests/conftest.py

```diff
@@ -201,6 +201,15 @@ def run_function_from_file(item, params=None):
 
     # Override environment variables for the subprocess
     env = os.environ.copy()
+    env.update(
+        {
+            # Add the tests path to the Python path so tests can import helpers
```

Comment:
Does this not work already? I am pretty sure we do this now?

Reply:
It might happen implicitly by running the tests from the root directory 🤔

Reply:
(I don't have any issue with this, since I think we assume this behavior elsewhere, just curious the specific issue you ran into)